### PR TITLE
Fix example command for getting pod name

### PR DESCRIPTION
**Pull request by nungster**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For pull requests on Chinese localization, set Base Branch to release-1.14
>
> For pull requests on Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by zacharysarah** [comment]
@nungster 👋 Please [sign the CLA](https://github.com/kubernetes/website/pull/15695#issuecomment-518731731) in order to continue.

File: content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ The job name and pod name are different.
 
 ```shell
 # Replace "hello-4111706356" with the job name in your system
-pods=$(kubectl get pods --selector=job-name=hello-4111706356 --output=jsonpath={.items[].metadata.name})
+pods=$(kubectl get pods --selector=job-name=hello-4111706356 --output=jsonpath='{.items[].metadata.name}')
```
Comment (xiangpengzhao) [inline content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md]: Actually, I get the same result by both commands. So I wonder what issue you are trying to fix :) Maybe wanting to get pod name list?

**Comment by zacharysarah** [comment]
@nungster Thanks for this PR. In order to proceed, we need more information in the description. What exactly are you trying to fix, and why?

Please feel free to `/reopen` this PR when you're ready to continue.

/close

**Pull request closed without merging**